src/caches/rosenbrock_caches.jl

```diff
@@ -95,7 +95,7 @@ function alg_cache(alg::Rosenbrock32,u,rate_prototype,uEltypeNoUnits,uBottomElty
   J,W = iip_generate_W(alg,u,uprev,p,t,dt,f,uEltypeNoUnits)
   tmp = zero(rate_prototype)
   atmp = similar(u, uEltypeNoUnits)
-  tab = Rosenbrock32ConstantCache(real(uBottomEltypeNoUnits),identity,identity,nothing,nothing,nothing)
+  tab = Ros32ConstantCache(real(uBottomEltypeNoUnits))
```

Comment:
Similar to my comment above, I think we should use something a bit more meaningful than `Ros32ConstantCache`.